keras_hub/src/models/gemma3/gemma3_causal_lm_preprocessor.py

```diff
@@ -14,10 +15,6 @@
 from keras_hub.src.utils.tensor_utils import preprocessing_function
 from keras_hub.src.utils.tensor_utils import strip_to_ragged
 
-START_OF_IMAGE_TOKEN = "<start_of_image>"
-IMAGE_PLACEHOLDER_TOKEN = "<img>"
-END_OF_IMAGE_TOKEN = "<end_of_image>"
-
 
 @keras_hub_export("keras_hub.models.Gemma3CausalLMPreprocessor")
```

Comment:
Not for this PR, but looks like our prompts, responses setup will not work for multi-turn conversations.  We should consider how we want that to work in the future.